lib/user.py

```diff
@@ -195,6 +235,7 @@ def get_changes(self, feed=None, limit=None, heartbeat=None, style=None,
         r.raise_for_status()
```

Comment:
Perhaps log an error if the changes feed is empty. If self.changes_data is not None, log an INFO, letting the test know the user changes cache will be reset